src/components/Composer/index.js

```diff
@@ -160,6 +151,11 @@ class Composer extends React.Component {
             document.addEventListener('drop', this.dragNDropListener);
             this.textInput.addEventListener('paste', this.handlePaste);
             this.textInput.addEventListener('wheel', this.handleWheel);
+
+            // selection will be at start (0,0) - so we need to update it to be at the
```

Comment:
```suggestion
            // Selection will be at start (0,0) - so we need to update it to be at the
```

We prefer comments starting with a Capital letter.